### PR TITLE
setup cypress configuration

### PR DESCRIPTION
**Pull request by maria-martinezr**

### CODE REVIEW

File: cypress/support/e2e.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (holgiosalos) [inline cypress/support/e2e.js]: EOF

File: cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (holgiosalos) [inline cypress/support/commands.js]: EOF

File: cypress/e2e/1-getting-started/todo.cy.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```
Comment (holgiosalos) [inline cypress/e2e/1-getting-started/todo.cy.js]: this file should not be part of the PR, the following paths should be included in the .gitignore file, as explained in [step 3.4:](https://github.com/testing-community/cypress-training-doc#3-instalaci%C3%B3n-de-cypress)
```
cypress/e2e/1-getting-started/*
cypress/e2e/2-advanced-examples/*
```

File: cypress/support/e2e.js (hunk 1 above)
Comment (maria-martinezr) [reply cypress/support/e2e.js]: Resolved!

File: cypress/support/commands.js (hunk 2 above)
Comment (maria-martinezr) [reply cypress/support/commands.js]: Resolved!

File: cypress/e2e/1-getting-started/todo.cy.js (hunk 3 above)
Comment (maria-martinezr) [reply cypress/e2e/1-getting-started/todo.cy.js]: Resolved!

**Review by manuelq12 — APPROVED** [review]

**Review by holgiosalos — APPROVED** [review]
